teuthology/test/task/__init__.py

```diff
@@ -1,5 +1,9 @@
 from mock import patch, DEFAULT
 from pytest import raises
+try:
+    from typing import Any
```

Comment:
why is this required?

Reply:
It's required for the type hints (even if these are only comments for py2).